src/api.js

```diff
@@ -0,0 +1,40 @@
+const scoreSrc = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/NNFdrdYO5isLUHbaJxs8/scores/';
+const scores = document.querySelector('#score_list');
+let scoresArr = [];
+
+const gameScores = async () => {
+  const response = await fetch(scoreSrc)
+    .then((resp) => resp.json())
+    .then((data) => data.result)
+    .catch(() => 'error');
+  return response;
+};
+
+const displayScores = () => {
+  gameScores().then((resp) => {
+    if (typeof resp === 'object') {
+      scoresArr = Array.from(resp);
+      scores.innerHTML = '';
+      if (scoresArr.length > 0) {
+        scoresArr.forEach((e) => {
+          const scoresTemp = `<li class="score" >${e.user}: ${e.score}</li>`;
+          scores.innerHTML += scoresTemp;
+        });
+      }
+    }
+  });
+};
```

Comment:
- [ ] Kindly use async/await for asynchronous operations as it is the project's requirement. :pray:

Reply:
I have added async/await to that function, thank you.

Reply:
Well done! This issue is resolved. :clap: :clap: :clap:

Reply:
Thank you!